src/Facebook/GraphNodes/GraphAlbum.php

```diff
@@ -110,9 +100,9 @@ public function getDescription()
     }
 
     /**
-     * Returns profile that created the album.
+     * Returns the profile that created the album.
      *
-     * @return GraphUser|null
+     * @return GraphNode|null
```

Comment:
What about something like:
```php
    /**
     * Returns the profile that created the album.
     *
     * @param bool @isPage true if you want a GraphPage instance
     *
     * @return GraphUser|GraphPage|null
     */
    public function getFrom($isPage = false)
    {
        if (!$isPage) {
             return $this->getField('from');
        }

        // return a casted to a GraphPage instance
    }
```

I'm not really sur. But for now, all users get a GraphUser instance, so I think we should keep this behavior as the default, and provide a way to get a GraphPage. :)